File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/riyousha/GetRiyoushaAdminByEntityCapsuleDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.riyousha;

import java.io.Serializable;

import net.seijishikin.jp.normalize.common_tool.dto.FrameworkCapsuleDto;
import net.seijishikin.jp.normalize.manage.kanrensha.entity.RiyoushaAdminMasterEntity;

/**
 * マスタEntityから利用者SE権限編集用利用者取得条件Dto
 */
public class GetRiyoushaAdminByEntityCapsuleDto extends FrameworkCapsuleDto implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 利用者SE権限マスタEntity */
    private RiyoushaAdminMasterEntity masterEntity;

    /**
     * 利用者SE権限マスタEntityを取得する
     * 
     * @return 利用者SE権限マスタEntity
     */
    public RiyoushaAdminMasterEntity getMasterEntity() {
        return masterEntity;
    }

    /**
     * 利用者SE権限マスタEntityを設定する
     * 
     * @param masterEntity 利用者SE権限マスタEntity
     */
    public void setMasterEntity(final RiyoushaAdminMasterEntity masterEntity) {
        this.masterEntity = masterEntity;
    }

}

File: app/front/src/components/main/dto/riyousha/getRiyoushaAdminByEntityCapsuleDto.ts
import { FrameworkCapsuleDto, type FrameworkCapsuleDtoInterface } from "seijishikin-jp-normalize_common-tool";
import { RiyoushaAdminMasterEntity, type RiyoushaAdminMasterEntityInterface } from "../../entity/riyoushaAdminMasterEntity";

interface GetRiyoushaAdminByEntityCapsuleDtoInterface extends FrameworkCapsuleDtoInterface {

    /** 利用者SE権限マスタEntity */
    masterEntity: RiyoushaAdminMasterEntityInterface;

}

class GetRiyoushaAdminByEntityCapsuleDto extends FrameworkCapsuleDto implements GetRiyoushaAdminByEntityCapsuleDtoInterface {

    /** 利用者SE権限マスタEntity */
    masterEntity: RiyoushaAdminMasterEntityInterface;

    constructor() {
        super();

        this.masterEntity = new RiyoushaAdminMasterEntity();
    }
}

export { type GetRiyoushaAdminByEntityCapsuleDtoInterface, GetRiyoushaAdminByEntityCapsuleDto }
